packages/react-native-select-pro/src/components/select/select.hooks.ts

```diff
@@ -7,6 +7,7 @@ import { ITEM_HEIGHT, MAX_HEIGHT_LIST } from '../../constants/styles';
 import { getSize, isSectionOptionsType } from '../../helpers';
 import { getReducedSectionData } from '../../helpers';
 import { selectedOptionResolver } from '../../helpers';
+import { getSectionOptionsIndexes } from '../../helpers/get-section-options-indexes';
```

Comment:
import { getSectionOptionsIndexes } from '../../helpers' and you can merge 7, 8, 9 and 10 line to one import